<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1137</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Bell MT&quot;;
border-color: rgb(130, 159, 255);
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1091</width>
       <height>643</height>
      </rect>
     </property>
     <attribute name="label">
      <string>milestone1-page1</string>
     </attribute>
     <widget class="QGroupBox" name="state_city">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>421</width>
        <height>621</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(189, 255, 230);</string>
      </property>
      <property name="title">
       <string>state/city</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>261</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>68</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>150</y>
         <width>251</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="business">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>631</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(189, 255, 230);</string>
      </property>
      <property name="title">
       <string>Business</string>
      </property>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>561</width>
         <height>561</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1091</width>
       <height>643</height>
      </rect>
     </property>
     <attribute name="label">
      <string>milestone2-page2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1137</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
